Src/spin.h

```diff
@@ -27,6 +27,7 @@ typedef struct Lextok {
 	unsigned short	ntyp;	/* node type */
 	short	ismtyp;		/* CONST derived from MTYP */
 	int	val;		/* value attribute */
+	ValueKind constValKind;
```

Comment:
Doc, for consistency.

Reply:
ok

Reply:
Do we have any doc for our (N7) changes that I should edit? or start a new changelog?

Reply:
Doc in the code is still not added.
As for changelog - I think that only a single changelog should be maintaned. Changes may be reported once they are fully implemented.